3600/2/commands/check_number.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'redis'
+require './identifiers'
+
+# :nodoc:
+module CheckNumber
+  include Identifiers
+  def start!(*)
+    if registred?
+      respond_with :message, text: 'You have already registered! Tap /checkin to continue'
+    else
+      save_context :register_from_message
+      respond_with :message, text: 'Please, enter your own Rubizza-number'
+    end
+  end
+
+  def register_from_message(*words)
+    respond_with :message, text: add_number(words)
+  end
+
+  private
+
+  def add_number(number)
+    redis = Redis.new
+    check_registaration(redis, number.first.to_i)
+  end
+
+  def check_registaration(redis, number)
+    if redis.get(number) || session.key?(:number)
```

Comment:
you can avoid using if else if you make return blabla if

Reply:
Smth like this?: return 'You have already registred' if redis.get(number) || sessionkey?(:number).
How many returns can be in one method?

Reply:
as you wish, but it should be pretty clear for everyone.

Reply:
I rewrote the method yesterday...